scripts/update_VM_to_full_CIL.sh

```diff
@@ -30,7 +35,4 @@ else
    cd ./CIL-Demos
 fi
 
-python -m pip install --user nbstripout
-nbstripout --install
```

Comment:
are you sure about this? Not having it causes no end of trouble. Of course, having it does cause some annoyance as well.

Reply:
They are installed when we install SIRF-Exercises, but probably you refer to the second line.

Reply:
oops. indeed!